BetterBags.toc

```diff
@@ -9,22 +9,24 @@
 ## X-License: MIT
 ## X-Curse-Project-ID: 942432
 ## X-Wago-ID: aNDmy96o
-## OptionalDeps: LibStub, Masque, CallbackHandler-1.0, Ace3, LibSharedMedia-3.0, _DebugLog, ConsolePort, Pawn, WagoAnalytics, GW2_UI, ElvUI, DevTool
+## OptionalDeps: LibStub, Masque, CallbackHandler-1.0, Ace3, LibSharedMedia-3.0, _DebugLog, ConsolePort, Pawn, WagoAnalytics, GW2_UI, ElvUI, DevTool, LibWindow-1.1, LibDataBroker-1.1, LibUIDropDownMenu, AceGUI-3.0-SharedMediaWidgets
```

Comment:
OptionalDeps are only needed when the deps are external to this addon. Please undo these (and feel free to remove Ace3, LSB, and CBH).